packages/react-stylesheets/etc/react-focus.api.md

```diff
@@ -0,0 +1,91 @@
+## API Report File for "@fluentui/react-focus"
```

Comment:
I think there might have been a bad copy of this file because I don't think this should live here.